src/foraging_gui/Foraging.py

```diff
@@ -2049,7 +2052,7 @@ def _Timer(self,Time):
     
     def _set_metadata_enabled(self, enable: bool):
         '''Enable or disable metadata fields'''
-        self.AnimalName.setEnabled(enable)
+        #self.AnimalName.setEnabled(enable)
```

Comment:
instead of commenting out this line, can you delete it?

Reply:
Sure! Done!